src/views/self/view_teleport.cpp

```diff
@@ -82,6 +84,178 @@ namespace big
+			{
+				for (auto id : BlipsMC)
+				{
+					ImGui::Spacing();
+					if (ImGui::Selectable(id.name))
+					{
+						g_fiber_pool->queue_job([=] {
+							teleport::to_blip((int)id.blipsprites);
+						});
+					}
+				}
+			}
+			ImGui::TreePop();
+		}
+
+		if (ImGui::TreeNode("Singleplayer"))
+		{
+			if (ImGui::CollapsingHeader("Safehouses"))
+			{
+				for (auto ID : spSafehouses)
+				{
+					ImGui::Spacing();
+					if (ImGui::Selectable(ID.name))
+					{
+						g_fiber_pool->queue_job([=] {
+							teleport::to_coords(ID.Coords);
+						});
+					}
+				}
+			}
+			ImGui::Text("  "); //two spaces
+			ImGui::SameLine();
+			if (ImGui::CollapsingHeader("Airfields"))
+			{
+				for (auto id : airfields)
+				{
+					ImGui::Spacing();
+					if (ImGui::Selectable(id.name))
+					{
+						g_fiber_pool->queue_job([=] {
+							teleport::to_coords(id.Coords);
+						});
+					}
+				}
+			}
+			ImGui::Text("  "); //two spaces
+			ImGui::SameLine();
+			if (ImGui::CollapsingHeader("Locations"))
+			{
+				for (auto id : outLocs)
+				{
+					ImGui::Spacing();
+					if (ImGui::Selectable(id.name))
+					{
+						g_fiber_pool->queue_job([=] {
+							teleport::to_coords(id.Coords);
+						});
+					}
+				}
+			}
+
+
+			if (ImGui::CollapsingHeader("Shops"))
+			{
+				ImGui::Text("  "); //two spaces
+				ImGui::SameLine();
+				if (ImGui::CollapsingHeader("Clothing"))
+				{
+					for (auto cloth : clothings)
+					{
+						ImGui::Spacing();
+						if (ImGui::Selectable(cloth.name))
+						{
+							g_fiber_pool->queue_job([=] {
+								teleport::to_coords(cloth.Coords);
+							});
+						}
+					}
+				}
+				ImGui::Text("  "); //two spaces
+				ImGui::SameLine();
+				if (ImGui::CollapsingHeader("Ammunation"))
+				{
+					for (auto id : ammunation)
+					{
+						ImGui::Spacing();
+						if (ImGui::Selectable(id.name))
+						{
+							g_fiber_pool->queue_job([=] {
+								teleport::to_coords(id.Coords);
+							});
+						}
+					}
+				}
+				ImGui::Text("  "); //two spaces
+				ImGui::SameLine();
+				if (ImGui::CollapsingHeader("Car Stuff"))
+				{
+					for (auto id : carsstuff)
+					{
+						ImGui::Spacing();
+						if (ImGui::Selectable(id.name))
+						{
+							g_fiber_pool->queue_job([=] {
+								teleport::to_coords(id.Coords);
+							});
+						}
+					}
+				}
+			}
+			if (ImGui::CollapsingHeader("Misc"))
+			{
+				ImGui::Spacing();
+				
+				if (ImGui::CollapsingHeader("Activities"))
+				{
+					for (auto id : activitiestele)
+					{
+						ImGui::Text("  "); //two spaces
+						ImGui::SameLine();
+						if (ImGui::Selectable(id.name))
+						{
+							g_fiber_pool->queue_job([=] {
+								teleport::to_coords(id.Coords);
+							});
+						}
+					}
+				}
+
+				for (auto id : misctele)
+				{
+					if (ImGui::Selectable(id.name))
+					{
+						g_fiber_pool->queue_job([=] {
+							teleport::to_coords(id.Coords);
+						});
+					}
+				}
+			}
+
+			ImGui::TreePop();
+		}
+
```

Comment:
These strings should be moved to the translation repository, however this should be done as a last step.

Reply:
Okay found out how to add to translation repo going to do it for the strings now